### ISSUE TITLE
Colon-pair details in REPL look buggy

### ISSUE THREAD

**Issue by fecundf** [perl6/roast#507]
https://docs.perl6.org/type/Pair says "Colon pairs can be chained without a comma to create a List of Pairs. Depending on context you may have to be explicit when assigning colon lists." - and follows with examples, which led me to experiments, that raise more questions. I would think all 5 of these should say the same thing but instead give 3 different answers. Does the doc need an addition to describe this? Is it a problem with Rakudo Star 2018.10?
```
$ perl6
To exit type 'exit' or '^D'
> (:a1:b2:c3)
(a1 => True b2 => 1 c3 => 1)
> (:a1,:b2,:c3)
(a1 => True b2 => True c3 => True)
> :a1,:b2,:c3
(a1 => True b2 => True c3 => True)
> :a1:b2:c3
a1 => True
> :a1 :b2 :c3
a1 => True
```

**Comment by JJ** [comment]
Maybe this is an issue for perl6/doc and not roast? Should I transfer it?

**Comment by lizmat** [comment]
Yeah.  Afaik, the colonpairs are currently working by design (with which I don't always agree, but that's another matter).
